src/examples/override_credentials.yml

```diff
@@ -17,7 +17,7 @@ usage:
         - aws-cli/setup:
             aws-access-key-id: AWS_ACCESS_KEY_ID_BLUE
             aws-secret-access-key: AWS_SECRET_ACCESS_KEY_BLUE
```

Comment:
Don't these need to be replaced with `_` too?

Reply:
Good catch. I changed it